linux/ibus-keyman/tests/testfixture.h

```diff
@@ -0,0 +1,17 @@
+#ifndef __TESTFIXTURE_H__
+#define __TESTFIXTURE_H__
+
+#include <glib-object.h>
+#include <glib.h>
+#include <ibus.h>
+#include "ibusimcontext.h"
+
+#define KEYMAN_F24_KEYCODE_OUTPUT_SENTINEL 194  // 0xC2
```

Comment:
Is this to keep in sync with [OrderedOutputDevice.cpp](https://github.com/keymanapp/keyman/pull/13234/files#diff-d2a722b3d353e80063dfb7a13d8ef538a64cb00a2574e4fc25bb7a2f4d57324b) ?

Reply:
yes, as well as `engine.c`. I added TODO comments to all three locations.